india_compliance/gst_india/overrides/transaction.py

```diff
@@ -1063,13 +1068,15 @@ def set(self, doc):
         self.doc = doc
         is_sales_transaction = doc.doctype in SALES_DOCTYPES
 
-        if is_overseas_doc(doc) and is_sales_transaction:
+        if is_sales_transaction and is_overseas_doc(doc):
             self.set_for_overseas()
             return
 
         self.gst_accounts = get_all_gst_accounts(self.doc.company)
         has_gst_accounts = any(
-            row.account_head in self.gst_accounts for row in self.doc.taxes
+            row.account_head in self.gst_accounts
```

Comment:
This needs to be handled differently. We can add validation for inconsistency between tax template and item taxes